tst/working/quick/nongeneric-classical.tst

```diff
@@ -0,0 +1,17 @@
+# test some non-generic cases
```

Comment:
What about calling this file `NonGenericClassical.tst`? The other test files also use camel case for their names.

Reply:
also fine by me.

Alice just pointed out to me that we should also test (representatives of) maximal subgroups to make sure they are *not* "recognized" incorrectly as their parent.